templates/miq-template.yaml

```diff
@@ -446,6 +524,8 @@ objects:
             value: "${POSTGRESQL_MAX_CONNECTIONS}"
           - name: POSTGRESQL_SHARED_BUFFERS
             value: "${POSTGRESQL_SHARED_BUFFERS}"
+          - name: POSTGRESQL_CONFIG_DIR
```

Comment:
I feel like this is a lot less likely to change now that we are specifying it at build time of our PG image.

@fbladilo Do you think people are likely to rebuild the image to change this?